runtime/doc/treesitter.txt

```diff
@@ -1366,6 +1368,19 @@ LanguageTree:language_for_range({range})
 LanguageTree:named_node_for_range({range}, {opts})
     Gets the smallest named node that contains {range}.
 
+    Parameters: ~
+      • {range}  (`Range4`) `{ start_line, start_col, end_line, end_col }`
+      • {opts}   (`table?`) A table with the following fields:
+                 • {ignore_injections}? (`boolean`, default: `true`) Ignore
+                   injected languages
+
+    Return: ~
+        (`TSNode?`)
+
+                                               *LanguageTree:node_for_range()*
+LanguageTree:node_for_range({range}, {opts})
```

Comment:
Do people actually like `named_` variants of all these functions, or would it make sense to make "named vs anon" a parameter?

Reply:
That sounds good to me! I don't have a strong preference either way.